src/ocean/io/select/protocol/task/TaskSelectTransceiver.d

```diff
@@ -35,7 +35,11 @@ class TaskSelectTransceiver
     import ocean.stdc.posix.sys.uio: iovec, readv;
     import ocean.stdc.posix.sys.socket: setsockopt;
     import ocean.stdc.posix.netinet.in_: IPPROTO_TCP;
-    import core.sys.linux.sys.netinet.tcp: TCP_CORK;
+
+    static if (__VERSION__ >= 2000 && __VERSION__ < 2073)
```

Comment:
This is why the comment for me it's really redundant, we should go to the source and just grep for `__VERSION__` when we drop support for a compiler and see them case by case, comments are not really reliable...